rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -269,11 +275,14 @@ public ResponseEntity<CollectionModel<EntityModel<Project>>> getProjectsForUser(
         boolean isSearchByType = CommonUtils.isNotNullEmptyOrWhitespace(projectType);
         boolean isSearchByGroup = CommonUtils.isNotNullEmptyOrWhitespace(group);
         boolean isNoFilter = false;
+        boolean isAllProjectAdded=false;
         String queryString = request.getQueryString();
         Map<String, String> params = restControllerHelper.parseQueryString(queryString);
         List<Project> sw360Projects = new ArrayList<>();
-        Map<String, Set<String>> filterMap = new HashMap<>();
         if (luceneSearch) {
+            Map<String, Set<String>> filterMap = getFilterMap(version, projectResponsible, state, clearingStatus,
+                    additionalData);
```

Comment:
Please move `group`, `projectType` and `tag` to `getFilterMap()` as well so it can be used by non-lucene search as well.